extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -65,9 +84,25 @@ export class AmpSelector extends AMP.BaseElement {
       this.element.setAttribute('aria-disabled', 'true');
     }
 
+    const kbSupportMode = this.element.getAttribute('keyboard-support');
```

Comment:
This definitely needs an experiment as we are writing code alongside existing functionality and this could totally break in ways we cant foresee. Also until we test a new feature and make sure it works well on production guarding by experiment is always a good choice. Forgetting to cleanup experiments is not an excuse for not having one.